client/components/command-pallette/wpcom-command-pallette.tsx

```diff
@@ -69,16 +69,23 @@ interface CommandInputProps {
 function CommandInput( { isOpen, search, setSearch }: CommandInputProps ) {
 	const commandMenuInput = useRef< HTMLInputElement >( null );
 	const _value = useCommandState( ( state ) => state.value );
+	const sanitizedValue = useMemo( () => {
+		const removedQuotesValue = _value.replace( /"/g, '' ); // Remove double quotes from any selected items before processing input
```

Comment:
Should we use `escapeAttribute` from `@wordpress/escape-html`? https://developer.wordpress.org/block-editor/reference-guides/packages/packages-escape-html/#escapeattribute

Or how do we match the sanitization the library is already applying, to make sure our value exactly matches what's rendered on the page?

Also, what other characters might cause problems?

Reply:
> Also, what other characters might cause problems?
Yesterday reviewing the PR I found that the double quotes character is problematic. The rest worked fine.